### DISCUSSION TITLE
OpenAI version compatibility

### DISCUSSION THREAD

**Discussion by cbornet** [run-llama/llama_index#8788] [General]
It seems that now llama-index depends on openai >=1.0 and at the same time it depends on langchain that depends on openai < 1.0
So there will probably be some unexpected failures there ?

**Comment by logan-markewich** [comment]
Yea if you are using langchain components, there will be some rough edges. I would either swap your langchain components with llama-index equivalents where possible, or use an older version of llamindex until langchain catches up
